test/integration/types/external_calls.spec.ts

```diff
@@ -95,14 +114,29 @@ contract Main {
 
         a.add(4);
 
+        payable(0).some();
+        payable(0).other();
+
         Some memory s = Some(Lib.w, fExtPtr);
 
         s.intFn();
         s.extFn();
+
```

Comment:
Could you add a test for calling `LibAddr.some(payable(0x0))`?

Reply:
Addressed in 4ea38d2